### PR TITLE
Added custom criterion and optimizer in classification.py

### LINKED ISSUE #3: Add a custom criterion and optimizer support for utils/classification.py
Adding support for choosing custom loss functions and optimizers would allow users more flexibility while working out with their experiments.

You have to edit the `train_model()` function for this:

A solution for this issue would look somethinig like this:

`train_model(model=model, train_dataloader=train_dataloader, test_dataloader=test_dataloader, ...... optimizer=torch.optim.<Optimizer>(), criterion=torch.nn.<Criterion>)`

### LINKED ISSUE #4: Add a custom criterion and optimizer support for utils/segmentation.py
Adding support for choosing custom loss functions and optimizers would allow users more flexibility while working out with their experiments.

You have to edit the `train_model()` function for this:

A solution for this issue would look somethinig like this:

`train_model(model=model, train_dataloader=train_dataloader, test_dataloader=test_dataloader, ...... optimizer=torch.optim.<Optimizer>(), criterion=torch.nn.<Criterion>)`

### PR DESCRIPTION
**Pull request by akshaankhan2004**

### CODE REVIEW

**Comment by ishan121028** [comment]
Thanks for this, can you also update the Readme.md file for this.
